### DISCUSSION TITLE
Data hierarchy with csv files

### DISCUSSION THREAD

**Discussion by emmanuel-ruffio** [binjr/binjr#151] [General]
Hi,

I'm trying to use Binjr to visualize large time series of temperature, luminosity, etc... I'm working with CSV files.
I'm wondering if there is a way to specify a hierarchy of data like I saw on the Binjr website with JRDS source:

1. Services
2. System
3.     >Load
4.     >Memory
5.          >RAM
6.          >etc
7. Network

Is the hierarchy available with csv files ? Would be nice if it was based on the data name specified in the first line of the file:
Services, System/Load, System/Memory/RAM,

Best regards,

**Comment by fthevenet** [comment]
Hi,

There exist no hierarchy explicitly expressed in CSV files for binjr to use, I'm afraid.

In the example you mention with JRDS, the hierarchy is provided by the JRDS source itself. Same thing for sources like Netdata, for instance.
